pkg/gofr/datasource/sql/sql_test.go

```diff
@@ -156,6 +156,15 @@ func Test_NewSQLMock(t *testing.T) {
 	assert.NotNil(t, mockMetric)
 }
 
+func Test_NewSQLMockWithConfig(t *testing.T) {
+	dbConfig := DBConfig{Dialect: "dialect", HostName: "hostname", User: "user", Password: "password", Port: "port", Database: "database"}
+	db, mock, mockMetric := NewSQLMocksWithConfig(t, &dbConfig)
+	assert.NotNil(t, db)
```

Comment:
declaration can be separated from assert statements - need to give an empty line between 161 and 162